### PR TITLE
Add sheet for policy vs. trend charts

### LINKED ISSUE #36: New data sheet for policy vs trend charts
To allow the charting of state-wide policy against the data, we need to generate a new data source (gdoc spreadsheet) which contains the follow time series:

1. Raw cases per million 
2. Spline cases per million 
3. Positivity

over the last 90 days. This could be inserted into the current "main site" sheet, but I would recommend a new one for organization.

### PR DESCRIPTION
**Pull request by smacpher**

### CODE REVIEW

File: covid/transform.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
 from covid.transform_utils import generate_lag_column_name_formatter_and_column_names
 from covid.transform_utils import generate_lags
 
+# Define miscellaneous constants.
+ONE_MILLION = 1_000_000
```
Comment (lucasmbrown) [inline covid/transform.py:20]: 👍

File: covid/transform.py [hunk 2]
```diff
@@ -135,6 +137,11 @@
 )
 CDC_CRITERIA_6A_14_DAY_MAX_PERCENT_POSITIVE = "CDC Criteria 6A"
 
+# Policy vs. Trend charts data fields.
+POLICY_VS_TREND_RAW_CASES_PER_MILLION = "Policy vs. Trend Cases per Million (Raw)"
```
Comment (lucasmbrown) [inline covid/transform.py]: Do we need `Policy vs. Trend` repeated in all the titles? (This is an honest question!)

File: covid/transform.py [hunk 3]
```diff
@@ -914,6 +954,25 @@ def transform_covidtracking_data(covidtracking_df):
...
+
+        # Calculate 3DCS cases per million.
+        covidtracking_df.loc[(state,), POLICY_VS_TREND_3DCS_CASES_PER_MILLION] = (
+            covidtracking_df.loc[(state,), NEW_CASES_3DCS_FIELD].values
+            / state_population
+        ) * ONE_MILLION
+
+        # Calculate positivity 3DCS.
+        covidtracking_df.loc[
+            (state,), POLICY_VS_TREND_3DCS_POSITIVITY
```
Comment (lucasmbrown) [inline covid/transform.py:910]: Do we need to copy the `FRACTION_POSITIVE_NEW_TESTS_3DCS_FIELD` column into `POLICY_VS_TREND_3DCS_POSITIVITY`, or can we just use the former?

**Review by lucasmbrown — COMMENTED** [review]
Looks great! Left a couple small questions.

**Comment by devinhunt** [comment]
Everything looks good to me. Only thing that comes to mind is that we need to make sure this sheet is updated nightly along with the rollup. Can we get this live asap?

**Pull request merged**